public/templates/patientSignUp.html

```diff
@@ -61,92 +66,85 @@
-              /> -->
-            </div>
-
-            <!--           <div class="field-wrap">
-            <label>
-              Email Address<span class="req">*</span>
-            </label>
-            <input type="email"required autocomplete="off"/>
-          </div> -->
-          </form>
-        </div>
-
-        <div id="login">
-          <h1>Welcome Back!</h1>
-
-          <form action="/" method="post">
-            <div class="field-wrap">
-              <label> Email Address<span class="req">*</span> </label>
-              <input type="email" required autocomplete="off" />
+          <ul class="tab-group">
+            <li class="tab active"><a href="#physical">Physical Health</a></li>
+            <li class="tab"><a href="#mental">Mental Health</a></li>
+            <li class="tab"><a href="#emotional">Emotional Health</a></li>
+            <li class="tab"><a href="#talk">Spiritual Health</a></li>
+          </ul>
+          <!-- ***** Patient Topic Selection End ***** -->
+          
+          <!-- ***** Tab Content Start ***** -->
+          <div class="tab-content">
+
+            <!-- ***** Sign Up Area Start ***** -->
+            <div>
+              <h1 class="formText">Fill The Form To Get Matched!</h1>
+              <form action="/" method="post">
+                <!-- ***** Top Row Start ***** -->
+                <div class="top-row">
+                  <!-- Input field for first name -->
+                  <div class="field-wrap">
+                      <label>First Name<span class="req">*</span> </label>
+                    <input type="text" name="firstName" id="firstName"  autocomplete="off" required/>
+                  </div>
+                  <!-- Input field for last name -->
+                  <div class="field-wrap">
+                    <label>Last Name<span class="req">*</span> </label>
+                    <input type="text" name="lastName"  id="lastName"  autocomplete="off" required/>
+                  </div>
+                </div>
+                <!-- ***** Top Row End ***** -->
+                <!-- Input field for description of issue -->
+                <div class="field-wrap">
+                  <textarea
+                    id="textArea"
+                    name="paragraph_text"
+                    placeholder="Please give a short description of your issue."
+                    cols="50"
+                    rows="10"></textarea>
+                  <button type="submit" id="button" class="button button-block" onclick ="formSubmit">
+                    <span style="color:#fff;">Match Me!</span>
+                  </button>
+                </div>
+              </form>
             </div>
+            <!-- ***** Sign Up Area End ***** -->
+
+            <!-- ***** Login Area Start ***** -->
+            <!-- CONFUSION: THE BELOW CODE SEEMS TO IMPACT THE WAY THE ABOVE FORM IS RENDERED BUT I DON'T KNOW WHY THIS IS THE CASE WHEN THE BELOW CODE IS FOR A LOGIN FORM NOT USED ON THIS PAGE -->
```

Comment:
Okay, so I was able to resolve this. There was a CSS selector in prSignUp.css that applied a "display:none" on the last  child of the div ".tab-content". When I deleted this login code, what was happening was that the sign up form was the last child and ended up being hidden. So what I did was change the CSS rule to do a display:none specifically on divs with an id of login: 

.tab-content  > #login{
display:none
}

I deleted the login form entirely from patientSignUp.html because it's unnecessary, but this style ^ is helpful for the other file that is linked to prSignUp.css (i.e. prSignUp.html).

Side Note: Apparently, there are some essential styles from prSignUp.css for patientSignUp.html to render properly so I can't eliminate the link to this stylesheet in that html file.